DNN Platform/Library/Common/Globals.cs

```diff
@@ -3620,101 +3707,143 @@ public static string UserProfilePicRelativeUrl(bool includeCdv)
-                string FileName = File.Substring(CurrentDirectory.FullName.Length);
+                string fileName = file.Substring(currentDirectory.FullName.Length);
                 if (strExtensions.IndexOf(strExtension, StringComparison.InvariantCultureIgnoreCase) != -1 || string.IsNullOrEmpty(strExtensions))
                 {
-                    arrFileList.Add(new FileItem(FileName, FileName));
+                    arrFileList.Add(new FileItem(fileName, fileName));
                 }
             }
 
             return arrFileList;
         }
 
-        [Obsolete("This method has been deprecated. Replaced by GetSubFolderPath(ByVal strFileNamePath As String, ByVal portaId as Integer).. Scheduled removal in v11.0.0.")]
+        /// <summary>
+        /// Gets the subfolder path for a give filename path.
+        /// </summary>
+        /// <param name="strFileNamePath">The filename full path.</param>
+        /// <returns>The subfolder name.</returns>
+        [Obsolete("This method has been deprecated. Replaced by GetSubFolderPath(ByVal strFileNamePath As String, ByVal portaId as Integer). Scheduled removal in v10.0.0.")]
         public static string GetSubFolderPath(string strFileNamePath)
         {
             // Obtain PortalSettings from Current Context
-            PortalSettings _portalSettings = PortalController.Instance.GetCurrentPortalSettings();
-            string ParentFolderName = null;
-            if (IsHostTab(_portalSettings.ActiveTab.TabID))
+            var portalSettings = PortalController.Instance.GetCurrentPortalSettings();
+            string parentFolderName = null;
+            if (IsHostTab(portalSettings.ActiveTab.TabID))
             {
-                ParentFolderName = HostMapPath.Replace("/", "\\");
+                parentFolderName = HostMapPath.Replace("/", "\\");
             }
             else
             {
-                ParentFolderName = _portalSettings.HomeDirectoryMapPath.Replace("/", "\\");
+                parentFolderName = portalSettings.HomeDirectoryMapPath.Replace("/", "\\");
             }
 
             string strFolderpath = strFileNamePath.Substring(0, strFileNamePath.LastIndexOf("\\") + 1);
 
-            return strFolderpath.Substring(ParentFolderName.Length).Replace("\\", "/");
+            return strFolderpath.Substring(parentFolderName.Length).Replace("\\", "/");
         }
 
-        [Obsolete("This function has been obsoleted: Use Common.Globals.LinkClick() for proper handling of URLs. Scheduled removal in v11.0.0.")]
-        public static string LinkClickURL(string Link)
+        /// <summary>
+        /// Gets a LinkClick url for tracking purposes.
+        /// </summary>
+        /// <param name="link">The actual link the LinkClick handler should point to.</param>
+        /// <returns>The formatted LinkClick url.</returns>
+        [Obsolete("This function has been obsoleted: Use Common.Globals.LinkClick() for proper handling of URLs. Scheduled removal in v10.0.0.")]

```

Comment:
Why did this change to v10 instead of v11?

Reply:
Addressed in latest commit, thanks.